content/matplotlib/concepts/pyplot/terms/scatter/scatter.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: '.scatter()'
+Description: 'Creates a scatter plot of x vs. y values.'
+Subjects:
+ - 'Data Science'
+ - 'Data Visualization'
+Tags:
+ - 'Graphs'
+ - 'Libraries'
+ - 'Matplotlib'
+CatalogContent:
+ - 'learn-python-3'
+ - 'paths/computer-science'
+---
+
+The **`.scatter()`** method in the matplotlib library is used to draw a scatter plot, showing a relationship between variables.
+
+## Syntax
+
+```pseudo
+matplotlib.pyplot.scatter(x, y, s, c, marker, cmap, norm, vmin, vmax, alpha, linewidths, edgecolors, plotnonfinite)
+```
+
+Both 'x' and 'y' parameters are required, and represent float or array-like objects. Other parameters are optional and vary plot features like marker size and/or color.
+
+`.scatter()` takes the following arguments:
+
+- `x` and `y`: Positional arguments of type float or array.
+- `s`: A float or an array (of size equal to x or y) specifying marker size.
+- `c`: An array or list specifying marker color.
+- `marker`: The marker style.
+- `cmap`: A Colormap instance used to map scalar data to colors.
+- `norm`: Normalization method used to scale scalar data to the [0, 1] range before mapping. Linear scaling is default.
+- `vmin` and `vmax`: Define the data range that the colormap covers (if norm is not specified).
```

Comment:
```suggestion
- `vmin` and `vmax`: Sets the data range for the colormap (if norm is not specified).
```

Reply:
Please be careful not repurpose descriptions in other documentation. All entry content should be original.